app/models/post.rb

```diff
@@ -1197,11 +1199,26 @@ def cannot_permanently_delete_reason(user)
   end
 
   def mentions
```

Comment:
Probably worth making this name clearer? What are mentions? user records? user ids? user names?